openmp/runtime/src/z_Linux_util.cpp

```diff
@@ -878,10 +881,19 @@ void __kmp_create_worker(int gtid, kmp_info_t *th, size_t stack_size) {
     KMP_SYSFAIL("pthread_create", status);
   }
 
-#ifdef LIBOMP_HAVE_LINUX_PTHREAD_SETNAME
+#if defined(LIBOMP_HAVE_PTHREAD_SETNAME_NP) ||                                 \
+    defined(LIBOMP_HAVE_PTHREAD_SET_NAME_NP)
   // Rename worker threads for improved debuggability
   if (!KMP_UBER_GTID(gtid)) {
+#if defined(__linux__)
     pthread_setname_np(handle, "openmp_worker");
+#elif defined(__FreeBSD__) || defined(__OpenBSD__)
```

Comment:
a question; why in some places do you check `KMP_OS_OPENBSD` and in other places `__OpenBSD__`?

Reply:
That's a good point. I have fixed that.